### DISCUSSION TITLE
wan口ipv6获取

### DISCUSSION THREAD

**Discussion by smomop** [coolsnowwolf/lede#9555] [R68S / 电犀牛专区]
想请问一下，wan口如何获取ipv6地址，我并不想实现lan口分配ipv6地址，主要用途就是使用wan口获取的v6地址添加一个dns解析，但是找了很多方法并不能实现

**Comment by dnslin** [comment]
lan口一样操作的

**Comment by github-h** [comment]
你应该是这个意思吧
![图片](https://user-images.githubusercontent.com/49114731/173992452-d70e80e3-4ccd-4bbf-a05d-3354bf5cc1ea.png)
![图片](https://user-images.githubusercontent.com/49114731/173992496-c3c94f8c-f0ba-40ab-a276-e1ddbe07a43d.png)

那么只用关闭lan口的v6功能即可  具体操作可以看#9540
